app/client/src/modules/private-room/components/room-messages/room-messages.component.tsx

```diff
@@ -68,64 +56,51 @@ export const RoomMessagesComponent: React.FC<Props> = ({
     setMessages([]);
     setYPivot(0);
 
-    const removeOnMessage = onProxy(
-      Event.MESSAGE,
-      ({ id, accountId, message, color }) => {
-        addMessage({
-          id,
-          accountId,
-          message,
-          color,
-        });
-      },
-    );
+    const removeOnMessage = onProxy(Event.MESSAGE, ({ id, accountId, message, color }) => {
+      addMessage({
+        id,
+        accountId,
+        message,
+        color,
+      });
+    });
 
-    const removeOnWhisperMessage = onProxy(
-      Event.WHISPER_MESSAGE,
-      ({ id, accountId, message, color }) =>
-        addMessage({
-          id,
-          accountId,
-          message,
-          color,
-          backgroundColor: 0xc1bfbf,
-        }),
+    const removeOnWhisperMessage = onProxy(Event.WHISPER_MESSAGE, ({ id, accountId, message, color }) =>
+      addMessage({
+        id,
+        accountId,
+        message,
+        color,
+        backgroundColor: 0xc1bfbf,
+      })
     );
 
-    const removeOnSystemMessage = onProxy(
-      Event.SYSTEM_MESSAGE,
-      ({ message }) => {
-        addMessage({
-          id: ulid(),
-          accountId: null,
-          username: "System",
-          message,
-          usernameColor: 0xffffff,
-          color: 0xffffff,
-          backgroundColor: 0xdba935,
-          messageColor: 0xffffff,
-        });
-        console.log(`System: ${message}`);
-      },
-    );
+    const removeOnSystemMessage = onProxy(Event.SYSTEM_MESSAGE, ({ message }) => {
+      addMessage({
+        id: ulid(),
+        accountId: null,
+        username: "System",
+        message,
+        usernameColor: 0xffffff,
+        color: 0xffffff,
+        backgroundColor: 0xdba935,
+        messageColor: 0xffffff,
+      });
+      console.log(`System: ${message}`);
+    });
 
     return () => {
       removeOnMessage();
       removeOnWhisperMessage();
       removeOnSystemMessage();
     };
-  }, [room?.id, addMessage, getUser, onProxy, setMessages, setYPivot]);
```

Comment:
addMessage, setMessage and setYPivot are called inside, the one causing the bug is getUser because it's not called and if we check whats going on with that function, we can check that the `rooms?.users` is part of the deps, so definitely getting out the getUser will fix the issue. Also, run prettier with `yarn run prettier:write` on the project @josealmon